app/javascript/controllers/clipboard_controller.js

```diff
@@ -0,0 +1,17 @@
+import Clipboard from 'stimulus-clipboard'
+
+export default class extends Clipboard {
+  connect() {
+    super.connect()
+    console.log('Do what you want here.')
```

Comment:
don't know whether you want to keep that logging

Reply:
ah, removed the other one, but forgot about this one. thanks!